src/solver/conv_hip_implicit_gemm_mlir_cpp_bwd.cpp

```diff
@@ -49,9 +70,31 @@ bool ConvHipImplicitGemmMlirCppBwd::IsApplicable(const ConvolutionContext& ctx)
     // Future: MLIR will support non-default layouts.
     if(!ctx.IsLayoutDefault())
         return false;
-    if(ctx.Is3d())
+    // Future: MLIR will support 3d convolution
+    if(!ctx.Is2d())
+        return false;
+    // Future: MLIR will support multiple data types
+    if(!ctx.IsFp32())
+        return false;
+    // Below: Generic checks between this solver and ConvHipImplicitGemmBwdDataV1R1
```

Comment:
[Recommendation] If/when `ConvHipImplicitGemmBwdDataV1R1` change, the comment will become invalid. I would recommend removing trivial comments.

Reply:
That's true, will get rid of them.